### ISSUE TITLE
Navtab inside navtab pane - BUG

### ISSUE THREAD

**Issue by xv1t** [twbs/bootstrap#24280]
Navtabs inside a navtab pane is not correct working in the 4 version. 

But in the BS3 working good!

In the tab **Profile** inserted a navtab with one element

Actions step
1. Select **Dropdown/fat**
2. Click **Profile**

But pane "Dropdown/fat" was not hided!

![screenshot](http://i.imgur.com/FlnYyWz.png)

This code in the [jsfiddle](https://jsfiddle.net/xv1t/8hv77eLp/2/)

```html
<div class="container-fluid">
  <h2>
  Nav tabs
  </h2>
  <div id="navtabs1">
<ul class="nav nav-tabs" id="myTab" role="tablist">
  <li class="nav-item">
    <a class="nav-link active" id="home-tab" data-toggle="tab" href="#home" role="tab" aria-controls="home" aria-expanded="true">Home</a>
  </li>
  <li class="nav-item">
    <a class="nav-link" id="profile-tab" data-toggle="tab" href="#profile" role="tab" aria-controls="profile">Profile</a>
  </li>
  <li class="nav-item dropdown">
    <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
      Dropdown
    </a>
    <div class="dropdown-menu">
      <a class="dropdown-item" id="dropdown1-tab" href="#dropdown1" role="tab" data-toggle="tab" aria-controls="dropdown1">@fat</a>
      <a class="dropdown-item" id="dropdown2-tab" href="#dropdown2" role="tab" data-toggle="tab" aria-controls="dropdown2">@mdo</a>
    </div>
  </li>
</ul>
<div class="tab-content" id="myTabContent">
  <div class="tab-pane fade show active" id="home" role="tabpanel" aria-labelledby="home-tab">Home Content</div>
  <div class="tab-pane fade" id="profile" role="tabpanel" aria-labelledby="profile-tab">
    <!-- Start nav tabs 2 -->
    <div id="navtabs2">
      <ul class="nav nav-tabs">
        <li class="nav-item">
          <a class="nav-link active"  id="profile-tab2" data-toggle="tab" href="#profile2" role="tab" aria-controls="profile2">Profile 2</a>
        </li>
      </ul>
      <div class="tab-content">
       <div class="tab-pane active"  id="profile2" role="tabpanel" aria-labelledby="profile-tab2">
         deepest navtabs2 pane
       </div>
      </div>
    </div>
   <!-- end navtab2 -->
  </div>
  <div class="tab-pane fade" id="dropdown1" role="tabpanel" aria-labelledby="dropdown1-tab">Dropdown fat</div>
  <div class="tab-pane fade" id="dropdown2" role="tabpanel" aria-labelledby="dropdown2-tab">Dropdown mdo</div>
</div>
  </div>
</div>
```

**Comment by XhmikosR** [comment]
Probably the same issue fixed by #23087.

/CC @Johann-S

**Comment by xv1t** [comment]
I took the latest version and happy :)
**XhmikosR** Thanks, its worked.
